packages/babel-types/src/definitions/flow.js

```diff
@@ -250,15 +250,15 @@ defineType("NumberTypeAnnotation", {
 });
 
 defineType("ObjectTypeAnnotation", {
-  visitor: ["properties", "indexers", "callProperties"],
+  visitor: ["properties", "indexers", "callProperties", "exact"],
```

Comment:
`visitors` should only contain `Node` properties, since it represents the properties which should be visited by babel-traverse.
What you need is `builder`, which represents the parameters passed to the `t.objectTypeAnnotation` function.